strings-csharp/HexToByteArray/Tests/Test.cs

```diff
@@ -4,85 +4,66 @@ namespace Tests
 {
     public class Test
     {
-        private readonly ConvertHex _convertHex;
         private const string HexString = "FF008000";
-        public Test()
-        {
-            _convertHex = new ConvertHex();
-        }
+        private readonly byte[] _expectedResult = { 255, 0, 128, 0 };
+        private readonly byte[] _expectedEmptyResult = new byte[0];
+
 
         [Fact]
         public void WhenValidHexPassed_ThenConverionSuccessful()
         {
-            byte[] expectedResult = { 255, 0, 128, 0 };
-            var result = _convertHex.FromHexString(HexString);
+            var result = ConvertHex.FromHexString(HexString);
 
-            Assert.Equal(expectedResult, result);
+            Assert.Equal(_expectedResult, result);
         }
 
         [Fact]
         public void WhenInvalidFormatHexPassed_ThenConverionFail()
         {
-            var expectedResult = "FormatException";
-            try
-            {
-                var result = _convertHex.FromHexString("Hello");
-            }
-            catch (Exception ex)
-            {
-                Assert.Equal(expectedResult, ex.GetType().Name);
-            }
+            var ex = Assert.Throws<FormatException>(() => ConvertHex.FromHexString("Hello"));
+
+            Assert.Equal("FormatException", ex.GetType().Name);
```

Comment:
The generic Assert.Throws<T> already verifies that the thrown exception is of type <T>, so this line can be removed. We also don't need to store the return of throws in a local variable.

Reply:
Done. Thanks!